DDCore/include/DD4hep/Shapes.h

```diff
@@ -829,6 +945,27 @@ namespace dd4hep {
     Trap& setDimensions(double z, double theta, double phi,
                         double h1, double bl1, double tl1, double alpha1,
                         double h2, double bl2, double tl2, double alpha2);
+
+    /// Accessor: phi value
+    double phi() const                     { return access()->GetPhi()*dd4hep::deg;    }
```

Comment:
@MarkusFrankATcernch  - IMHO we need to avoid double conversion, e.g. converting everything to radians (e.g. G4 definition) and then back to degrees (e.g. ROOT definition). We will definitely see some precision loss.